Solutions/CiscoSEG/Parsers/CiscoSEGEvent.txt

```diff
@@ -0,0 +1,49 @@
+// Usage Instructions: 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name as CiscoSEGEvent and specifying Legacy Category.
+// This function maps Cisco SEG Consolidated Event Logs to normalization schema (https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema).

```

Comment:
This documentation link has en-us locale that is causing the Validation to fail. Please update the link.

Reply:
updated